sys/net/network_layer/fib/fib.c

```diff
@@ -412,14 +424,17 @@ int fib_get_next_hop(kernel_pid_t *iface_id,
     fib_entry_t *entry[count];
 
     if( iface_id == NULL
-        || next_hop == NULL
         || next_hop_size == NULL
-        || next_hop_flags == NULL
-        || dst == NULL) {
+        || next_hop_flags == NULL) {
```

Comment:
You might want to add some parens here, too, while you're at it, but I don't insist.